NST/src/main/java/com/ebay/service/logger/formats/loader/CustomLoggerFormatManager.java

```diff
@@ -16,7 +16,7 @@
 import com.ebay.service.logger.platforms.HarLogger;
 import com.ebay.service.logger.platforms.IosLogger;
 
-public final class CustomLoggerFormatManager {
+public class CustomLoggerFormatManager {
```

Comment:
Why make this non-final?

Reply:
We cannot extend this class because the constructor is private. Having the class be final was causing problems with Mocking. I chose to remove the final specifier in order to support testing knowing the class could not be extended with the private constructor.